src/Components/CardboardListCallout/CardboardListCallout.tsx

```diff
@@ -127,7 +127,16 @@ const CardboardListCallout = <T extends unknown>({
                     <PrimaryButton
```

Comment:
please add jest tests for this component to validate this new functionality. You can look at `ViewerConfigUtility.ts` for examples.

Reply:
can you point me where exactly in ViewerConfigUtility.ts? or did you mean like jest test as in PropertyInspectorModel.test.ts?

Reply:
yeah. like that. if you look at the `ViewerConfigUtility.test.ts` you'll see the tests I wrote for that file.

Reply:
Will be separate PR, working on it